osf/models/institution_storage_region.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from osf.models.base import BaseModel
+
+
+class InstitutionStorageRegion(BaseModel):
+
+    institution = models.ForeignKey('Institution', on_delete=models.CASCADE)
+    storage_region = models.ForeignKey('addons_osfstorage.Region', on_delete=models.CASCADE)
+    is_preferred = models.BooleanField(default=False)
```

Comment:
For CR: There is no constraints to limit only one preferred region for each institution. See next two comments for more details.

Reply:
Yes, this is exactly what I was looking for, will update the constraints (and affected code)